frontend/packages/console-dynamic-plugin-sdk/src/validation/ExtensionValidator.ts

```diff
@@ -29,13 +75,24 @@ export const collectCodeRefData = (extensions: Extension[]) =>
 export const findWebpackModules = (
   compilation: webpack.Compilation,
   exposedModules: ConsolePluginBuildMetadata['exposedModules'],
+  basePath?: string,
```

Comment:
This isn't a public API so I think this argument should be mandatory - any reason why it's marked as optional?

Treating it as non-optional will simplify the code below.